es-space-mgmt/main/default/lwc/reservationTile/__tests__/reservationTile.test.js

```diff
@@ -0,0 +1,123 @@
+import { createElement } from 'lwc';
+import ReservationTile from 'c/reservationTile';
+
+const RESERVATION_CONTACT = require('./data/contactReservation.json');
+
+const RESERVATION_LEAD = require('./data/leadReservation.json');
+
+describe('c-reservation-tile', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('render reservationTile for Lead', () => {
+        const element = createElement('c-reservation-tile', {
+            is: ReservationTile
+        });
+        element.reservation = RESERVATION_LEAD;
+        document.body.appendChild(element);
+        return Promise.resolve().then(() => {
```

Comment:
Would be great to check if you can add here and in other places a few new lines to increase readability.